src/stores/widgets/SpectralLineQueryWidgetStore.ts

```diff
@@ -137,6 +140,14 @@ export class SpectralLineQueryWidgetStore extends RegionWidgetStore {
         this.queryUnit = queryUnit;
     };
 
+    @action setIntensityLimitState = () => {
```

Comment:
can we rename this to `toggleIntensityLimit` and change `isApplyingIntensityLimit` to `intensityLimitEnabled`

Reply:
renamed.